doc/developer/building-frr-for-debian12.rst

```diff
@@ -0,0 +1,127 @@
+Debian 12
+========================================
```

Comment:
Please align `=` with the end of `Debian 12`.

Reply:
OK